src/bci_build/package/helm/README.md.j2

```diff
@@ -0,0 +1,107 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+This image packages the [Helm](https://helm.sh/) binary, a [Kubernetes](https://kubernetes.io/) package manager that helps manage Kubernetes applications.
+
+## Description
+
+[Helm](https://helm.sh/) is a tool for managing [Kubernetes](https://kubernetes.io/) charts, which are packages of pre-configured Kubernetes resources. This container image simplifies the deployment and use of Helm by packaging it into a ready-to-use container.
+
+
+## Usage
+
+To pull Helm container image from the SUSE registry and start an interactive shell:
+
+```ShellSession
+podman run --rm -it --entrypoint /bin/sh {{ image.pretty_reference }}
+```
+
+### Connecting Helm to the Host's Kubernetes Cluster
+
+To interact with a Kubernetes cluster running on the host, mount the Kubernetes configuration file (`kubeconfig`) into the container (use `--net=host` flag to allow container to use host’s network):
+
+```ShellSession
+podman run --rm -it --net=host -v /path/to/kubeconfig:/root/.kube/config --entrypoint /bin/sh {{ image.pretty_reference }}
+```
+
+### Using Helm in the Container
```

Comment:
This entire section feels weird if we can run as follows:

`podman run <helm-image> helm <subcommand>`

We don't need to teach all Helm commands here as it is available in official docs.